platooning_strategic/include/platoon_strategic.h

```diff
@@ -512,12 +530,11 @@ namespace platoon_strategic
             const std::string MOBILITY_STRATEGY = "Carma/Platooning";
             const std::string OPERATION_INFO_TYPE = "INFO";
             const std::string OPERATION_STATUS_TYPE = "STATUS";
+            const std::string JOIN_REQUIREMENTS_TYPE = "JOIN_REQUIREMENTS";
```

Comment:
Is this another `MobilityOperation` sub-type? Let's label it as `OPERATION_JOIN_REQUIREMENTS_TYPE` for consistency with the elements above it if so.

Reply:
Fixed.